data_updates/Python/oecd_data.py

```diff
@@ -0,0 +1,137 @@
+
+import datetime
+import os
+import glob
+import numpy
+from numpy import NaN
+import pandas as pd
+from github import Github, InputGitTreeElement
+from django.conf import settings
+
+current_date = datetime.datetime.now()
+PURPOSE_CODE_TRENDS_URL = os.getenv('PURPOSE_CODE_TRENDS_URL') or "https://staging-ddw.devinit.org/api/export/1241/"
```

Comment:
I think the default can be added directly in the `getenv` function i.e
`os.getenv('PURPOSE_CODE_TRENDS_URL', 'https://staging-ddw.devinit.org/api/export/1241/')`

... do for the others as wells